dev-requirements.txt

```diff
@@ -6,7 +6,7 @@ pytest-cov
 pytest-rerunfailures
 pytest-timeout
 pytest-xdist
-pyyaml
+https://github.com/yaml/pyyaml/archive/master.zip#egg=pyyaml
```

Comment:
we might want to do that conditionnaly depending on the `python_version` ? With maybe a comment to revert this when a 3.7-supporting pyyaml is released ?

Reply:
Sounds good to me.